dashboard/app/api.go

```diff
@@ -104,11 +104,8 @@ func handleJSON(fn JSONHandler) http.Handler {
 		c := appengine.NewContext(r)
 		reply, err := fn(c, r)
 		if err != nil {
-			// ErrAccess is logged earlier.
-			if err != ErrAccess {
-				log.Errorf(c, "%v", err)
-			}
-			http.Error(w, err.Error(), http.StatusInternalServerError)
+			status := logErrorPrepareStatus(c, err)
```

Comment:
Does this change existing logic or not? If yes, what's the change?

Reply:
Currently we don't log ErrAccess b/c we already logged it.
Do we log it now?

Reply:
It was a part of checkClient() errors processing. Now that code returns fmt.Errorf() and we process everything here.